.gitignore

```diff
@@ -5,3 +5,4 @@
 /lib/logging/Makefile
 project.xcworkspace/
 xcuserdata/
+/lib/fles_libfabric/*~
```

Comment:
Suggest adding editor-specific ignores to ~/.gitignore_global instead of the project's .gitignore.